tests/k8s/bootstrap-w-conf.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
```

Comment:
that's a little bit of a mess. I see you added this file and the config. but you didn't add `w-conf` for clients. also you kept the old file as well. 
I prefer we won't push half-baked features to develop and we split it to a different pull request.